circle.yml

```diff
@@ -35,21 +35,21 @@ jobs:
       - run:
           name: Build
           command: |
-              cmake . -DBAZQUX=ON -DFEEDHQ=ON
-              make
-      - run:
-          name: Test
-          command: make test
-      - run:
-          name: Generate JUnit XML
-          command: |
-              mkdir -p /tmp/test-results/junit/
-              for f in *.gtester.log; do
-                  xsltproc -o /tmp/test-results/junit/$(basename $f .gtester.log).xml gtester-to-junit-4.xsl $f;
-              done
-          when: always
-      - store_test_results:
-          path: /tmp/test-results
-      - store_artifacts:
-          path: /tmp/test-results/junit
-          destination: test-results
+              meson builddir -DBAZQUX=ON -DFEEDHQ=ON
+              ninja
```

Comment:
I think you need to do `cd builddir` first?

Reply:
Not needed with meson